cli/pcluster/configure/easyconfig.py

```diff
@@ -167,76 +141,40 @@ def configure(args):
     automate_vpc = prompt("Automate VPC creation? (y/n)", lambda x: x == "y" or x == "n", default_value="n") == "y"
 
     vpc_parameters = _create_vpc_parameters(
-        vpc_label, aws_region_name, scheduler, scheduler_handler.max_cluster_size, automate_vpc_creation=automate_vpc
+        vpc_section, aws_region_name, scheduler, scheduler_handler.max_cluster_size, automate_vpc_creation=automate_vpc
     )
-    global_parameters = {
-        "name": "global",
-        "cluster_template": cluster_template,
-        "update_check": "true",
-        "sanity_check": "true",
-    }
-    aws_parameters = {"name": "aws", "aws_region_name": aws_region_name}
-    cluster_parameters = {
-        "name": cluster_label,
-        "key_name": key_name,
-        "vpc_settings": cluster_template,
-        "scheduler": scheduler,
-        "master_instance_type": master_instance_type,
-    }
+    cluster_parameters = {"key_name": key_name, "scheduler": scheduler, "master_instance_type": master_instance_type}
```

Comment:
Actually we are using the old pcluster_config to retrieve the default values to show to the user so I think it is better to keep the config consistent and change all the parameters only at the end, when collected from the user all the required information.

We cannot reset vpc_parameters before `_create_vpc_parameters`, because we need the old values to propose the defaults.

We are using these variables as "temporary" variables to save the user's choiches, reset the pcluster_config and then add the new values.
We could improve the code but it is not strictly related to my patch, so I'd leave for future.